src/tests/Loader/regressions/polyrec/polyrec.cs

```diff
@@ -63,24 +63,24 @@ public static void Start2()
     genmeth2<string>(ninsts);
   }
 
-  public static int Main(String[] args)
+  public static int Test(int threads, int insts)
```

Comment:
In a case like this, would it make sense to keep the `Main` that parses args and passes them to `Test`, which you've created, but also add a test function that can be marked with `[Fact]` to call it with the precise arguments the test case is using, e.g.,

```
[Fact]
public static int Test1()
{
   return Test(4, 50);
}
```

the idea being, if you want to run the standalone test manually with arbitrary arguments, you still can, but the test harness calling the `[Fact]` function will get the specific arguments.

Maybe for this one it doesn't matter, but I'm thinking maybe some of the JIT benchmarks might be like this.

(btw, unrelated, note that when you get to `src\tests\JIT\Performance\CodeQuality`, those are "as close as we can make them" to the copies in the dotnet/performance repo at `src\benchmarks\micro\runtime`)